src/runner/nunit.runner/Services/TestOptions.cs

```diff
@@ -44,5 +58,15 @@ public class TestOptions
         /// Creates a NUnit Xml result file on the host file system using PCLStorage library.
         /// </summary>
         public bool CreateXmlResultFile { get; set; }
+
+        /// <summary>
+        /// File path for the xml result file
+        /// Default is [LocalStorage]/NUnitTestOutput/TestResults.xml
+        /// </summary>
```

Comment:
If I'm reading right, I think this comment is out-dated, default is now just in localstorage? Looks like the behaviour was changed since the comment was written. ☺